### ISSUE TITLE
Large hand optimization

### ISSUE THREAD

**Issue by Clemdawgg1** [kleinfreund/balatrolator#11]
### Description

Be able to configure a card and set how many are in hand for high card duplication

### Motivation

Some hands reach hundreds of cards and inputting them individually is simply unfeasible. I tried verifying a WR hand claim and the hand size was 302/8. https://www.reddit.com/r/balatro/comments/1bs0e2c/rank_1_wr_largest_possible_score_71439e2665_no/

**Comment by kleinfreund** [comment]
@Clemdawgg1 Do you think a "Duplicate card" functionality added to both playing cards and jokers would allow people to input large hands more easily?

In the most simple terms, this would be a basic duplication, but it's also conceivable to open a dialog first where one enters the number of additional copies to create first. This way, one could easily duplicate a specific card with its entire configuration several times. Of course this wouldn't help with large hands that have more card variety. 

Also, this is related to #10.

**Comment by kleinfreund** [comment]
I'm closing this as done as I've added the card duplication feature I mentioned (see https://github.com/kleinfreund/balatrolator/issues/10#issuecomment-2573727564 for a few details).

**Comment by fetcheroo** [comment]
While a marginal improvement, I think an interesting addition could be an ability to group duplicates into a single card stack with a counter indicating their number.

Of course if it's all the same card (which is an overwhelmingly likely scenario for such large hands) and the only thing desired is to get the final score, just putting in 300 cards and scrolling past them is easy enough. But being able to switch properties of an entire stack of cards to consider what type of card is better to duplicate could be neat.

**Comment by kleinfreund** [comment]
I agree that the current duplicate functionality isn't well-suited for hands for late endless runs scoring really, really high scores (e.g. those with hundreds of held steel, red sealed kings and Baron/Mime).

I'm thinking to instead provide an additional input on joker and playing cards that is a numerical input for "Play N times" where you'd put in 300 to make a card behaves as if you'd duplicated it 299 times.

**Comment by fetcheroo** [comment]
Sounds like a good idea, will help devices with less screen space too!
